core/src/test/java/org/elasticsearch/search/searchafter/SearchAfterIT.java

```diff
@@ -225,6 +245,7 @@ private void assertSearchFromWithSortValues(String indexName, String typeName, L
                 requests.add(client().prepareIndex(INDEX_NAME, TYPE_NAME, Integer.toString(i)).setSource(builder));
             }
             indexRandom(true, requests);
+            ensureGreen();
```

Comment:
Ok I removed all the calls after indexRandom and keep the ones after createIndex. Does that make sense ?